extensions/PhabBugz/lib/Util.pm

```diff
@@ -251,10 +269,10 @@ sub request {
         $ua->default_header('Content-Type' => 'application/x-www-form-urlencoded');
     }
 
-    my $phab_api_key  = $params->{phabricator_api_key};
-    my $phab_base_uri = $params->{phabricator_base_uri};
-    ThrowUserError('invalid_phabricator_uri') unless $phab_base_uri;
-    ThrowUserError('invalid_phabricator_api_key') unless $phab_api_key;
+    my $phab_api_key = $params->{phabricator_api_key}
+      || ThrowUserError('invalid_phabricator_api_key');
```

Comment:
nit: The named operator would be better here. Seeing || makes me think you care about the return value.